### ISSUE TITLE
Error on Texturing node

### ISSUE THREAD

**Issue by Doritos777** [alicevision/meshroom#1342]
Thank you guys for this awesome program.

I was trying to figure out how to reduce the mesh resolution, so I added extra nodes for low resolution mesh like this screenshot below.

Meshing2, MeshResampling, MeshFiltering2 worked fine, 
But Texturing2 shows a error.
Maybe node setup is wrong? 
I didn't changed any parameters.

Hopefully It's just my lack of understanding of this program : )

**Screenshots**
`https://i.imgur.com/e6k0V6Q.jpg`
ps.
I tried to change the order of filtering/sampling but still same kind of issue.

**Desktop (please complete the following and other pertinent information):**
 - OS: Windows 10 1909
 - Meshroom version: Release 2021.1.0 (2020/02/26)
 - 
**Log**

> Program called with the following parameters:
 * angleHardThreshold = 90
 * bestScoreThreshold = 0.1
 * correctEV = 0
 * downscale =  Unknown Type "unsigned int"
 * fillHoles = 0
 * flipNormals = 0
 * forceVisibleByAllVertices = 0
 * imagesFolder = "i:/3dscans/MeshroomCache/DepthMap/e0b45024773b556ada89e55a44fa7048f1b50be1"
 * input = "i:/3dscans/MeshroomCache/Meshing/61a98875e30102481436d1b68bb17065ab189d2d/densePointCloud.abc"
 * inputMesh = "i:/3dscans/MeshroomCache/MeshFiltering/1684916cf1819ff5b98f4934d45a4d76010794c7/mesh.obj"
 * multiBandDownscale =  Unknown Type "unsigned int"
 * multiBandNbContrib =  Unknown Type "class std::vector<int,class std::allocator<int> >"
 * output = "i:/3dscans/MeshroomCache/Texturing/89b982f9a27034899c28ed2598ca3fc9b7f0c225"
 * outputTextureFileType = "png"
 * padding =  Unknown Type "unsigned int"
 * processColorspace = "sRGB"
 * subdivisionTargetRatio = 0.8
 * textureSide =  Unknown Type "unsigned int"
 * unwrapMethod = "Basic"
 * useScore = 1
 * useUDIM = 1
 * verboseLevel = "info"
 * visibilityRemappingMethod = "PullPush"

[21:42:51.586564][info] Load dense point cloud.
[21:42:51.765109][warning] Alembic LOAD: NO OBSERVATIONS_WITH_FEATURES: , mvg_visibilityFeatId: 0, mvg_visibilityFeatPos: 0, OBSERVATIONS_WITH_FEATURES flag: false
[21:42:52.771648][fatal] Cannot find image file coresponding to the view '122174840' in folder 'i:/3dscans/MeshroomCache/DepthMap/e0b45024773b556ada89e55a44fa7048f1b50be1/'.

**Comment by natowi** [comment]
Does the Mesh in MeshResampling look ok? Sometimes the Texturing node can fail when the Mesh is "oversimplified" or otherwise unusable.

Try to place MeshResampling between MeshFiltering and Texturing.

**Comment by Doritos777** [comment]
> Does the Mesh in MeshResampling look ok? Sometimes the Texturing node can fail when the Mesh is "oversimplified" or otherwise unusable.
> 

There was some dented area but I thought that's usable.

> Try to place MeshResampling between MeshFiltering and Texturing.
>
I tried same thing like I mentioned below the screenshot link, but still had same result.

I noticed default Simplication factor in MeshResampling was 0.5, so I switched to 0.9 since you mentioned "oversimplifed" could be the thing. 

Resampled mesh looked better but face normal is flipped here.
And still got same error on Texturing2 node.

https://i.imgur.com/q3iyVMS.jpg
